bin/benchmark_elbo.jl

```diff
@@ -12,7 +12,7 @@ const dat_dir = joinpath(Pkg.dir("Celeste"), "dat")
 srand(1)
 println("Loading data.")
 
-S = 100
+S = 3
```

Comment:
Could you leave this as `S = 100`, and `ElboDeriv.elbo` with `calculate_hessian=false`? We should make sure the run time for computing derivatives hasn't changed despite all the other changes to the code.